plan/2.0 Expand Training.md

```diff
@@ -38,338 +16,519 @@
-- Y1 - XX hours for roadmap development and content creation
+---
 
-## 2.6.2: Expand Secure Development thought leadership through blogs, webinars, podcasts
+## 2.7 Goal: Alternate delivery streams for content
 
-### Explanation: Many traditional channels are available to augment traditional classes such as pdocasts, blogs, and webinars. These are generally inexpensive and readily-accessioble tools to amplify the awareness of key concepts and techniques
+### Explanation: Sponsor existing or create alternate delivery streams for educational content
 
 ### Key Steps/Milestones
 
-- **M1**: Write 12 blogs supporting SIG & WG efforts
-- **M2**: Create and deliver 2 secure development webinars
-- **M3**: Create "Secure from the (open) Source" podcast and produce 12 episodes
+- **M1**: Establish a job description for an educational content developer.
 
-### Time & resource estimate
+  - [ ] Create a job description for an educational content developer
+  - [ ] Get feedback on the job description
+  - [ ] post opening and interview candidates
+  - [ ] onboard secdev contributors
 
-- Y1 - $35K to equip and produce podcast including graphics and music
-- Y2+ - $6k annual for ongoing poscast expenses
-- 60 volunteer hours to write 12 blogs - annual
-- 24 OSSF support hours to facilitate editing and publication of 12 blogs - annual
-- 48 volunteer hours to create and record 2 webinars - annual
-- 24 OSSF support hours to facilitate the recording, marketing, and publishing of 2 webinars - annual
-- 60 volunteer hours to record, produce, and publish 12 podcasts (audio and video) - annual
-- 12 OSSF support hours to facilitate podcast marketing and publication - annual
+- **M2**: Create a calendar of conferences where our content would be useful to train
 
-## 2.6.3: Expand Secure Development education with content to "training camps" and other groups
+  - [ ] Identify potential conferences where we want to present the materials and platform
+  - [ ] Contact conferences and apply for CFP / CFT
+  - [ ] Create a calendar of selected conferences
 
-### Explanation
+- **M3**: Creation of training slides
 
-### Key Steps/Milestones
+  - [ ] Create a "Bootcamp" slide deck for using as delivery method for in-person trainings
+  - [ ] Deliver two "Bootcamps" at LF-sponsored conferences based on existing Secure Development Fundamentals class
 
-- **M1**: Review existing content
-- **M2**: Generate modified classes
-- **M3**: train trainers
-- YEAR 2 - adjust for new content delivered during YEAR 1
-- **M4**: Dependent on 1.1 - Expand/improve existing content and training materials
-- **M5**: Dependent on 2.6 - Partnerships with underserved communities and alternate learning paths
+- **M4**: Creation of materials for other channels
 
-### Time & resource estimate
+  - [ ] Deliver twelve "thought leadership" secure development blogs
+  - [ ] Deliver 2 secure development OSSF webinars
```

Comment:
Is this two per language? if not this sounds like it will be introductory or universally applicable which may not provide as much value as a language or problem specific webinar as more fundamental secure development information already exists.